bootstrap/absolute.sh

```diff
@@ -26,6 +26,8 @@ if [ ! -f .pkg_timestamp ] || [ package.json -nt .pkg_timestamp ]; then
   npm update && > .pkg_timestamp
 fi
 
+export BABEL_CACHE_PATH=$(absolute_path)/.babel-cache.json
```

Comment:
IMHO, this might not be a good way because the boostrap code(shell script) should not have a dependency with JS code if possible. The babel works in JS layer. So, we can set the environment before babel runs.